app/assets/stylesheets/application.css.scss

```diff
@@ -110,6 +110,30 @@ body {
   background-position: center center;
 }
 
+.application-body-edoras{
+  background-color: black;
+  background-image: url('edoras.jpg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-minas_tirith{
```

Comment:
Selector `application-body-minas_tirith` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space